### ISSUE TITLE
net/freeradius: template does not put client shortname in double quotes

### ISSUE THREAD

**Issue by borisneubert** [opnsense/plugins#315]
OPNsense 17.7.5-amd64
FreeBSD 11.0-RELEASE-p12
OpenSSL 1.0.2l 25 May 2017

Had FreeRADIUS running. After changing Client IP Address or Network to 192.168.31.0/24, FreeRADIUS does not start any more. Changing it back to 192.168.31.1 does not help either.

Trying to start it from the Services | FreeRADIUS | General page or from the Dashboard gives me in the system.log:

```
Oct  8 12:35:31 opnsense configd.py: [6c44c135-6e77-4d4d-80bd-4b18a2708e9b] request FreeRADIUS status
Oct  8 12:35:38 opnsense configd.py: [9ef2b941-fc9a-4f87-8e83-f959be5d535c] request FreeRADIUS status
Oct  8 12:35:38 opnsense configd.py: [5a4deedb-321d-4966-b560-9fb9be80cf81] stopping FreeRADIUS
Oct  8 12:35:38 opnsense configd.py: [80170836-3497-4d8a-b39e-0b693a11c4e8] generate template OPNsense/Freeradius
Oct  8 12:35:39 opnsense configd.py: generate template container OPNsense/Freeradius
Oct  8 12:35:40 opnsense configd.py: [594f5459-ab28-4343-85c6-da8bd24db71c] starting FreeRADIUS
Oct  8 12:35:40 opnsense configd.py: [9e0884fc-0385-4fdc-9130-9e9c5e2dc67d] request FreeRADIUS status
```

The radius.log is touched but no entry is written (last entry is from previous run).

Running radiusd -X from the console shows the problem:

[code]/usr/local/etc/raddb/clients.conf[4]: Syntax error: Expected comma after 'Captive': Portal[/code]

The template that creates clients.conf does not put the shortname in double quotes. This is a problem for shortnames with blanks like "Captive Portal".

I suggest the following:
- Add double quotes to the template ./local/opnsense/service/templates/OPNsense/Freeradius/clients.conf
- Redirect the FreeRADIUS startup messages either to the system.log or the radius.log

Workaround: enter the double quotes in the GUI.

**Comment by mimugmail** [comment]
@borisneubert Thanks for the hint, never saw spaces in a shortname, but this should be fixed.
I added a fix to #313

**Comment by borisneubert** [comment]
Since we are on it: https://docs.opnsense.org/manual/how-tos/accounting.html suggests that the username should also be put in double quotes (section Add test users).

**Comment by mimugmail** [comment]
@borisneubert There's no need to put in double quotes, in user mgmt spaces are disallowed for user and pw. Would be better to change this in the docs ( CC @jschellevis  )

The fix for clients is now in master and will be released with 17.7.6 or 17.7.7

**Comment by fichtner** [comment]
Will be in 17.7.6, thanks everyone!
